features/step_definitions/filtering_steps.rb

```diff
@@ -331,6 +332,7 @@
     expect(page).to have_content('Register a magical spell')
   end
 
+  expect(page).to have_css("a[data-track-category='navFinderLinkClicked']")
```

Comment:
does the test check to see if the analytics work after live search?

Reply:
Any changes to the form, removed the tracking, so the test checks if the tracking is still available on form change.

Reply:
I have added a Jasmin test for this